### DISCUSSION TITLE
2.0.0-rc-2 - preparations for 2.0.0-final

### DISCUSSION THREAD

**Discussion by szpak** [gradle-nexus/publish-plugin#315] [Announcements]
### Notable changes:
- Add closeStagingRepositories and releaseStagingRepositories summary tasks - #236 (PR by @Vampire)
- Require Gradle 6.2+
- Ability to build itself with Gradle 8.5
- Multiple internal refactorings and improved idiomaticity (PRs by @TWiStErRob and @3flex)
- Drop kotlin-dsl plugin and remove redundant buildSrc - #295 and #301 (PRs by @3flex)
- Make the artifact ID explicit rather than relying on implicit naming - #255 (PR by @TWiStErRob)
- Massive dependency upgrades with Renovate

Full list of changes: https://github.com/gradle-nexus/publish-plugin/compare/v2.0.0-rc-1...v2.0.0-rc-2

Thanks to all the contributors to this release: @TWiStErRob, @3flex, @Goooler, @Vampire, @espertus and @renovate-bot ;-).

<hr /><em>This discussion was created from the release <a href='https://github.com/gradle-nexus/publish-plugin/releases/tag/v2.0.0-rc-2'>2.0.0-rc-2 - preparations for 2.0.0-final</a>.</em>

**Comment by markslater** [comment]
Do you have a schedule in mind for the release of 2.0.0?  I'm itching to get my hands on it!

**Reply by TWiStErRob** [reply]
Did you try rc-2? Everything works?

**Reply by markslater** [reply]
Yes, everything seems to work perfectly in 2.0.0-rc-2 for me on a sample project.  The only (minor) breaking change I encountered was `closeAndReleaseStagingRepository` becoming `closeAndReleaseStagingRepositories`.

**Reply by szpak** [reply]
@markslater  2.0.0-final has just been released: https://github.com/gradle-nexus/publish-plugin/releases/tag/v2.0.0
